spec/ndx-ophys-devices.extensions.yaml

```diff
@@ -1,8 +1,216 @@
 groups:
-- neurodata_type_def: TetrodeSeries
-  neurodata_type_inc: ElectricalSeries
-  doc: An extension of ElectricalSeries to include the tetrode ID for each time series.
+- neurodata_type_def: Indicator
+  neurodata_type_inc: Device
+  doc: Extends Device to hold metadata on the Indicator.
   attributes:
-  - name: trode_id
-    dtype: int32
-    doc: The tetrode ID.
+  - name: label
+    dtype: text
+    doc: Indicator standard notation.
+  - name: injection_location
+    dtype: text
+    doc: Injection brain region name.
+    required: false
+  - name: injection_coordinates_in_mm
+    dtype: float
+    shape:
+    - 3
+    doc: Indicator injection location in stereotactic coordinates (AP, ML, DV) mm
+      relative to Bregma.
+    required: false
+- neurodata_type_def: OpticalFiber
+  neurodata_type_inc: Device
+  doc: Extends Device to hold metadata on the Optical Fiber.
+  attributes:
+  - name: model
+    dtype: text
+    doc: Model of optical fiber.
+    required: false
+  - name: numerical_aperture
+    dtype: float
+    doc: Numerical aperture, e.g., 0.39 NA.
+    required: false
+  - name: core_diameter_in_um
+    dtype: float
+    doc: Core diameter in micrometers.
+    required: false
+- neurodata_type_def: ExcitationSource
+  neurodata_type_inc: Device
+  doc: Extends Device to hold metadata on the Excitation Source.
```

Comment:
```suggestion
  doc: Extends Device to hold metadata on the Excitation Source. Examples may include: x, y, or z.
```

Can you add onto this doc?

Reply:
Should I add examples of excitation sources or examples of the metadata that describes the excitation sources?

Reply:
The expanded illumination type is probably good enough to showcase this now